examples/Migration/CampaignManagement/CreateCompleteCampaignBothApisPhase2.php

```diff
@@ -0,0 +1,435 @@
+    }
+
+    /**
+     * Creates a campaign.
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param string $customerId the client customer ID without hyphens
+     * @param CampaignBudget $campaignBudget the campaign budget
+     * @return Campaign the newly created campaign
+     */
+    private static function createCampaign(
+        GoogleAdsClient $googleAdsClient,
+        string $customerId,
+        CampaignBudget $campaignBudget
+    ) {
+        $trueValue = new BoolValue(['value' => true]);
+        $falseValue = new BoolValue(['value' => false]);
+
+        $startDate = new StringValue(['value' => date('Ymd', strtotime('+1 day'))]);
+        $endDate = new StringValue(['value' => date('Ymd', strtotime('+1 month'))]);
+
+        $campaign = new Campaign([
+            'name' => new StringValue(['value' => 'Interplanetary Cruise #' . uniqid()]),
+            'advertising_channel_type' => AdvertisingChannelType::SEARCH,
+            // Recommendation: Set the campaign to PAUSED when creating it to prevent
+            // the ads from immediately serving. Set to ENABLED once you've added
+            // targeting and the ads are ready to serve.
+            'status' => CampaignStatus::PAUSED,
+            // Sets the bidding strategy and budget.
+            'manual_cpc' => new ManualCpc(),
+            'campaign_budget' => $campaignBudget->getResourceName(),
+            // Adds the network settings configured above.
+            'network_settings' => new NetworkSettings([
+                'target_google_search' => $trueValue,
+                'target_search_network' => $trueValue,
+                'target_content_network' => $falseValue,
+                'target_partner_search_network' => $falseValue
+            ]),
+            // Optional: Sets the start and end dates.
+            'start_date' => $startDate,
+            'end_date' => $endDate
+        ]);
+
+        // Creates a campaign operation.
+        $campaignOperation = new CampaignOperation();
+        $campaignOperation->setCreate($campaign);
+
+        // Issues a mutate request to add campaigns.
+        $campaignServiceClient = $googleAdsClient->getCampaignServiceClient();
+        $campaignResponse = $campaignServiceClient->mutateCampaigns(
+            $customerId,
+            [$campaignOperation]
+        );
+
+        $campaignResourceName = $campaignResponse->getResults()[0]->getResourceName();
+        $newCampaign = self::getCampaign($googleAdsClient, $customerId, $campaignResourceName);
+
+        printf("Added campaign named '%s'.%s", $newCampaign->getName()->getValue(), PHP_EOL);
+
+        return $newCampaign;
+    }
+
+    /**
+     * Gets a campaign.
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param string $customerId the client customer ID without hyphens
+     * @param string $resourceName the resource name of the campaign to retrieve
```

Comment:
From Phase 2 to final phase, your PHPDoc here is not aligned with the real parameter.
Same for getAdGroup

Reply:
Done.